app/assets/src/components/ui/controls/dropdowns/ThresholdFilterDropdown.jsx

```diff
@@ -3,14 +3,18 @@ import { forbidExtraProps } from "airbnb-prop-types";
 import PrimaryButton from "../buttons/PrimaryButton";
 import PropTypes from "prop-types";
 import RemoveIcon from "../../icons/RemoveIcon";
+import {
```

Comment:
Can we keep this drop down generic, please? 
Use the props as before and pass the values from outside.

Reply:
Well, it is called "ThresholdFilterDropdown". Why is it inappropriate to use a constant called THRESHOLD_METRICS if this is the level of abstraction we're operating at?

Reply:
I added this constant here because the props in all places where it was used for duplicated. IMO, we can reevaluate if we ever start using a threshold filter for metrics that are not related to the current ones. Right now, this use case is not even on the horizon, so I feel using props for data that never changes is misleading

Reply:
Because you cannot ever customize the metrics you want to use it with for different pages.
ThresholdFilterDropdown only means that you are applying a group of logical comparisons to a group of metrics.
I made it generic on purpose for easy reusability. 

What you are doing here is simply passing props, thorugh constants.

If you really want to make a default one, use default props to set defaults.
Also, please consolidate the two imports by defining something that equates to `MetricTextAndValue(THRESHOLD_METRICS)` in the `Metrics.js`.

Reply:
ok